### ISSUE TITLE
No Dark Theme Built From Git Pop!_OS

### ISSUE THREAD

**Issue by remlap** [celluloid-player/celluloid#682]
Overview Description: 

Just built from the Git on Pop!_OS 21.04 and I can not select a dark theme even though the option is ticked.

Actual Results: 

Only a light theme available.

Expected Results: 

Dark theme like in the Ubuntu repositories version.

Version: 

Built from Git.

**Comment by gnome-mpv** [comment]
That's weird. Dark theme should be the default.

Are you able to get dark theme working for any other GTK applications?

**Comment by remlap** [comment]
Yep and I build the release from the git too and it builds with a dark theme.

**Comment by jannuary** [comment]
Mainline uses GTK 4, so I'd assume Pop just doesn't ship a theme for that.
